src/components/theme/View/View.test.jsx

```diff
@@ -20,6 +20,7 @@ beforeAll(() => {
       ECONNREFUSED: () => <div className="ECONNREFUSED" />,
     },
   });
+  config.settings.publicURL = 'https://plone.org';
```

Comment:
Can we have a safer default? Maybe take RAZZLE_API_PATH, or a variant, as the default?

Reply:
This is only the test, the setting itself is in the config

Reply:
For the test is fine I guess. The config in the tests resets on every test.